src/ui/react/src/components/base/toolbar-buttons.js

```diff
@@ -31,12 +58,12 @@
 
             var toolbarButtons = this.filterExclusive(
                     buttons.filter(function(button) {
-                        return button && (AlloyEditor.Buttons[button] || AlloyEditor.Buttons[button.name]);
+                        return button && (AlloyEditor.Buttons[button] || AlloyEditor.Buttons[button.name] || AlloyEditor.Elements[button]);
```

Comment:
This is a bit weird. I'd just keep the `separator` inside `buttons` for now, even if it's not technically one. All the components that do build buttons are actually in there even though they're not actually buttons.

I'll simplify this for now and keep the hierarchy to a minimum. Once me work on moving to higher React versions we can put things anywhere and rely on dynamic imports or a smarter way to deal with this.

Reply:
Ok, I thought having an `Elements` section for the `separator` was more logic because it isn't a button, but now that this is clarified, I'll move it to the `buttons` directory. Thanks for the info!